person.rb

```diff
@@ -27,4 +29,9 @@ def of_age?
   def can_use_services?
     of_age? || @parent_permission
   end
+
+  def add_rental(date, book)
+    rental = Rental.new(date, book, self)
+    @rentals << rental
+  end
```

Comment:
- [ ] We don't need to add the same rental twice to the array. Kindly refactor this method and remove the code on line 35